frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetManageResources/ClusterSetManageResources.tsx

```diff
@@ -67,39 +76,95 @@ export function ClusterSetManageResourcesContent() {
     const deploymentDictionary = new Map<string | undefined, ClusterDeployment>()
     clusterDeployments?.forEach((deployment) => deploymentDictionary.set(deployment.metadata.name, deployment))
 
-    const [managedClusters, managedClusterSets] = useRecoilValue(
-        waitForAll([managedClustersState, managedClusterSetsState])
-    )
+    const clusters = useAllClusters()
+    const [managedClusterSets] = useRecoilValue(waitForAll([managedClustersState, managedClusterSetsState]))
+    const [clusterCurators] = useRecoilState(clusterCuratorsState)
+
```

Comment:
I'm not sure if `waitForAll` was ever needed... certainly looks like it shouldn't be now. What do you think?